shaping/input.go

```diff
@@ -37,6 +43,12 @@ type Input struct {
 	Language language.Language
 }
 
+// FontFeature sets one font feature.
+type FontFeature struct {
+	Tag   loader.Tag
+	Value uint32
+}
```

Comment:
Could you add a little more documentation about this type? A link to a list of standard font feature tags and common values would be sufficient. As it stands, I don't think many users will understand what this is.